cmake/OpenEXRConfig.h.in

```diff
@@ -17,9 +17,9 @@
 //
 
 #ifndef INCLUDED_OPENEXR_VERSION_H
-#define OPENEXR_VERSION_MAJOR @OpenEXR_VERSION_MAJOR@
```

Comment:
I'm curious what prompted this change? It should be benign, as I believe OPENEXR_VERSION_* and OpenEXR_VERSION_* have the same values. Not opposed to it, just wondering why it should change.

Reply:
When migrating to bazel we take and make bazel versions of these macros, and having them all consistently upper case seams cleaner for these kinds of external automation.